spec/support/vcr.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'vcr'
+require 'webmock'
+
+VCR.configure do |config|
+  config.ignore_localhost = true
+  config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
+  config.hook_into :webmock
+  config.configure_rspec_metadata!
+  config.allow_http_connections_when_no_cassette = false
```

Comment:
[This part](https://github.com/solidusio-contrib/solidus_paypal_marketplace/blob/main/spec/support/vcr.rb#L58-L59) would be useful to invalidate the cassettes.